src/custom/hooks/usePriceImpact/useFallbackPriceImpact.ts

```diff
@@ -65,14 +59,13 @@ export default function useFallbackPriceImpact({ abTrade, isWrapping }: Fallback
   }, [shouldCalculate])
 
   // Calculate the necessary params to get the inverse trade impact
-  const { parsedAmount, outputCurrency, ...swapQuoteParams } = useMemo(
-    () => ({
+  const { parsedAmount, outputCurrency, ...swapQuoteParams } = useMemo(() => {
+    return {
       ..._getBaTradeParams({ abTrade, sellToken, buyToken }),
       parsedAmount: _getBaTradeParsedAmount(abTrade, shouldCalculate),
-      validTo: lastQuote?.validTo,
-    }),
-    [abTrade, buyToken, lastQuote?.validTo, sellToken, shouldCalculate]
-  )
+      validTo: Math.round(Date.now() / 1000) + ms`30m`, // +30min - it's request to get price, so we don't need precise value for validTo
```

Comment:
Does this work?

I think this is adding milliseconds to seconds.

The [ms macro](https://github.com/knpwrs/ms.macro) will always give you the value in milliseconds.
And on the first part the date is being divided by `1000`

Maybe this is what you want?

```suggestion
      validTo: Math.round((Date.now() + ms`30m`)/ 1000) , // +30min - it's request to get price, so we don't need precise value for validTo
```

I would suggest also to create a constant such as DEFAULT_TIME_ADDED_TO_VALID_TO...ETC

Reply:
Thanks @alfetopito!
Fixed